shared/src/sys/out.rs

```diff
@@ -8,26 +8,25 @@
 //! Read more at https://github.com/rust-lang/rust/issues/73755.
 
 pub mod actor {
-    #[repr(C)]
     #[derive(Debug, Copy, Clone)]
+    #[repr(packed, C)]
     pub struct ResolveAddress {
         pub resolved: i32,
         pub value: u64,
     }
 }
 
 pub mod ipld {
-    #[repr(C)]
     #[derive(Debug, Copy, Clone)]
+    #[repr(packed, C)]
     pub struct IpldOpen {
```

Comment:
For all structs, can we add a note aimed as SDK implementers to explain that this is not necessarily the recommended argument order for SDK methods, but rather an alignment-optimised order? The first intuition is to expand structs to SDK function arguments honouring the order, which can lead to awkward APIs.